src/main/java/com/CezaryZal/api/body/BodySizeRepository.java

```diff
@@ -1,66 +1,45 @@
 package com.CezaryZal.api.body;
 
+import com.CezaryZal.api.body.entity.BodySize;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.query.Param;
 import org.springframework.stereotype.Repository;
-import org.springframework.transaction.annotation.Transactional;
 
-import javax.persistence.EntityManager;
-import javax.persistence.PersistenceContext;
-import javax.persistence.Query;
+import java.sql.Date;
 import java.time.LocalDate;
 import java.util.List;
+import java.util.Optional;
 
-@Transactional
 @Repository
-public class BodySizeRepository {
+public interface BodySizeRepository extends CrudRepository<BodySize, Long> {
 
-    @PersistenceContext
-    private EntityManager entityManager;
 
-    public BodySize findById (Long id){
-        return entityManager.find(BodySize.class, id);
-    }
+    @org.springframework.data.jpa.repository.Query(value = "select id from day where date =:inputDate and user_id =:userId", nativeQuery = true)
```

Comment:
strasznie brzydko wygląda ten import

Reply:
niestety to IDE :(
nie próbowałem, ale ręcznie nie wurzuca błedu